### ISSUE TITLE
(0 , _sync.go) is not a function

### ISSUE THREAD

**Issue by AlexBeauchemin** [auth0/lock#582]
Getting `Uncaught TypeError: (0 , _sync.go) is not a function` from core.js line 75

I'm using the latest version of auth0-lock (10.2.1) in chrome
I'm using rollup as a bundler (with commonjs and node-resolve plugins) https://github.com/AlexBeauchemin/people-iq
I'm using node 6.3.1

Here is a repo to reproduce (call is in app.jsx) 
[https://github.com/AlexBeauchemin/people-iq](https://github.com/AlexBeauchemin/people-iq)

Not sure if the error is coming from the bundling

**Comment by cristiandouce** [comment]
I see you changed to use auth0-lock-passwordless instead in that repo. Is this still an issue? Could you try again with the latest Lock version?

**Comment by AlexBeauchemin** [comment]
Hi, yes but auth0-lock-passwordless had the same problem. I switched to webpack instead of rollup as it is way more resilient with file structures and I had similar problems with a lot of other libraries.

I tried again though with a barebone project and it seems to be fixed. Same for auth0-lock-passwordless, so I guess I'm closing this. Thanks!

PS: For anyone finding this in the future, don't forget to exclude `node_modules/buffer-es6/**` in your rollup config if you are using `rollup-plugin-commonjs`
